### DISCUSSION TITLE
Update only some fields of patient via REST API

### DISCUSSION THREAD

**Discussion by andreiSeer** [dcm4che/dcm4chee-arc-light#4204] [General]
I noticed that the if a field is not on the body request, that info is erased from pacs. Example:

`{"00100020": {"vr": "LO", "Value": ["118621"] },"00100010": {"vr": "PN","Value": ["Testing JR"]}}
`

In this case  i only want to update the patient name, and not the other fields. What happens is that the name is updated but the other fields are erased. Is there a way to prevent that?

**Comment by vrindanayak** [comment]
> I noticed that the if a field is not on the body request, that info is erased from pacs. Example:
> 
> `{"00100020": {"vr": "LO", "Value": ["118621"] },"00100010": {"vr": "PN","Value": ["Testing JR"]}} `
> 
> In this case i only want to update the patient name, and not the other fields. What happens is that the name is updated but the other fields are erased. Is there a way to prevent that?

No, the complete dataset needs to be sent in the request payload.
